mobly/controllers/android_device_lib/snippet_client_v2.py

```diff
@@ -76,6 +78,23 @@
 _CALLBACK_DEFAULT_TIMEOUT_SEC = 60 * 2
 
 
+@dataclasses.dataclass
+class Config:
+  """A configuration class for configuring the snippet client.
```

Comment:
the am instrumentation options are passed to the server side right?
so this is not exactly "configuring the snippet client"?

Sometimes less is more :P

Reply:
Thanks. Removed it.